test/UnitTests/TestFramework.UnitTests/Assertions/AssertTests.IsInRange.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+
+using TestFramework.ForTestingMSTest;
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting.UnitTests;
+
+/// <summary>
+/// Unit tests for the <see cref="Assert.IsInRange{T}"/> methods.
+/// </summary>
+public partial class AssertTests : TestContainer
+{
+    #region IsInRange Tests
+
+    public void IsInRange_WithValueInRange_DoesNotThrow()
```

Comment:
Please also add some test for negative values in range (all negative and range spanning from negative to positive).